src/platform/suecreek/include/platform/memory.h

```diff
@@ -331,19 +334,28 @@
 #define SOF_MEM_RO_SIZE			0x8
 
 /* code loader */
-#define BOOT_LDR_TEXT_ENTRY_BASE	0xBE000000
+#define BOOT_LDR_TEXT_ENTRY_BASE	REEF_LOAD_BASE
 #define BOOT_LDR_TEXT_ENTRY_SIZE	0x400
 #define BOOT_LDR_LIT_BASE		(BOOT_LDR_TEXT_ENTRY_BASE + BOOT_LDR_TEXT_ENTRY_SIZE)
 #define BOOT_LDR_LIT_SIZE		0x400
 #define BOOT_LDR_TEXT_BASE		(BOOT_LDR_LIT_BASE + BOOT_LDR_LIT_SIZE)
-#define BOOT_LDR_TEXT_SIZE		0x800
+#define BOOT_LDR_TEXT_SIZE		0x1800
 #define BOOT_LDR_DATA_BASE		(BOOT_LDR_TEXT_BASE + BOOT_LDR_TEXT_SIZE)
 #define BOOT_LDR_DATA_SIZE		0x1000
 #define BOOT_LDR_BSS_BASE		(BOOT_LDR_DATA_BASE + BOOT_LDR_DATA_SIZE)
 #define BOOT_LDR_BSS_SIZE		0x100
 
 /* TODO: set this value */
-#define BOOT_LDR_MANIFEST_BASE		0x55aa55aa
+#define MAN_DESC_OFFSET			0x2000	/* from rimage/manifest.h */
+/*
+ * This is a temporary hack until rimage is updated to handle Sue Creek:
+ * currently the resuult of "xtensa-...-objdump -O binary sof-sue" is about
+ * 8250 bytes, reserve 9K for it and create a firmware image with something like
+ * dd if=/dev/zero bs=1 count=$(expr 9216 - $(stat --print=%s boot_ldr-sue.bin)) >> boot_ldr-sue.bin
+ * cat boot_ldr-sue.bin sof-sue.ri > /tmp/sue.img
```

Comment:
Sorry, can you also include the rimage updates here too so we dont need any hacks.

Reply:
That would take me some time. Should I do that now or after IPC?